### PR TITLE
Bugzilla 2647 - Update RAML “/oic/p” property

### PR DESCRIPTION
**Pull request by johnzhangjun**
Add "nct" (network connectivity type) as an optional property to "/oic/p"

### CODE REVIEW

File: schemas/oic.wk.p-schema.json [hunk 1]
```diff
@@ -95,6 +95,16 @@
           "maxLength": 64,
           "readOnly": true,
           "description": "Manufacturer's defined information for the platform. The content is freeform, with population rules up to the manufacturer"
+        },
+       "nct": {
```
Comment (mkettrick) [inline schemas/oic.wk.p-schema.json]: The Property Name in the CR is "mnnct" not "nct"  Which one is right?

File: schemas/oic.wk.p-schema.json [hunk 2]
```diff
@@ -95,6 +95,16 @@
...
+          "type": "array",		  
+          "items": {
+            "type": "integer",
+			"description": "The network connectivity type based on IANAIfType value as defined by: https://www.iana.org/assignments/ianaiftype-mib/ianaiftype-mib"
+          },
+          "minItems": 1,
+		  "readOnly": true,
```
Comment (mkettrick) [inline schemas/oic.wk.p-schema.json]: Should there be a range?  Are negative integers allowed?

File: schemas/oic.wk.p-schema.json (hunk 1 above)
Comment (johnzhangjun) [reply schemas/oic.wk.p-schema.json]: sorry, the property name should be changed to "mnnct" to align with the CR

File: schemas/oic.wk.p-schema.json (hunk 2 above)
Comment (johnzhangjun) [reply schemas/oic.wk.p-schema.json]: The property value follows IANAifType assigned numbers and IANAifType are positive integers from 1 to 294, it seems the numbers keep growing (https://www.iana.org/assignments/ianaiftype-mib/ianaiftype-mib)

**Pull request closed without merging**